client/src/App.jsx

```
@@ -1,17 +1,30 @@
+import Login from './components/admin/Login';
+import 'quill/dist/quill.snow.css';
 
 const App = () => {
   return (
     <div>
       <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/blog/:id" element={<Blog />} />
+        <Route path='/' element={<Home />} />
+        <Route path='/blog/:id' element={<Blog />} />
+        <Route path='/admin' element={true ? <Layout /> : <Login />}>
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)